backend/transcribee_backend/routers/document.py

```diff
@@ -129,20 +133,22 @@ def create_default_tasks_for_document(
     )
     session.add(align_task)
 
-    speaker_identification_task = Task(
-        task_type=TaskType.IDENTIFY_SPEAKERS,
-        task_parameters={},
-        document_id=document.id,
-        dependencies=[align_task],
-    )
-    session.add(speaker_identification_task)
+    if n_speakers != 0 and n_speakers != 1:
+        speaker_identification_task = Task(
+            task_type=TaskType.IDENTIFY_SPEAKERS,
+            task_parameters={"n_speakers": n_speakers},
+            document_id=document.id,
+            dependencies=[align_task],
+        )
+        session.add(speaker_identification_task)
 
 
 @document_router.post("/")
 async def create_document(
     name: str = Form(),
     model: str = Form(),
     language: str = Form(),
+    n_speakers: Optional[int] = Form(None),
```

Comment:
The validation could also be added here https://fastapi.tiangolo.com/tutorial/path-params-numeric-validations/#number-validations-greater-than-or-equal